fastlane/Fastfile

```diff
@@ -23,27 +20,13 @@ platform :ios do
       output_files: 'compile_commands.json,report.html,report.junit')
 
     if is_ci?
-      puts "Running on CI, coverage report will be generated by Danger"
+      puts "Running on CI, coverage report will be generated by travis"
```

Comment:
I'd write Travis instead of travis :)

Reply:
oh, yep I should have written `Travis` but I'm not making another commit/push/CI cycle just for that 😉